smoke-test/src/main/resources/minion-config/minion-config.yaml

```diff
@@ -22,3 +22,9 @@ telemetry:
 netmgt:
   traps:
     trapd.listen.interface: "0.0.0.0"
+
+karaf:
+  shell:
+    ssh:
+      host: "0.0.0.0"
+      port: 8201
```

Comment:
Looks like this is the default. Could remove it, s'pose it doesn't hurt though.

Reply:
Will remove `port` - but `host` definitely has to stay.